GitUI/UserControls/RevisionGrid/RevisionDataGridView.cs

```diff
@@ -299,9 +299,9 @@ private void OnCellPainting(object sender, DataGridViewCellPaintingEventArgs e)
             e.Handled = true;
         }
 
-        public void Add(GitRevision revision, RevisionNodeFlags types = RevisionNodeFlags.None)
+        public void Add(GitRevision revision, RevisionNodeFlags types = RevisionNodeFlags.None, bool addParentGraph = true)
```

Comment:
Ditto. Bonus points for xml-doc'ing everything on this method.

Reply:
Will update